bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/update/thing-update.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="mybmw:bev_rex">
+		<instruction-set targetVersion="4">
```

Comment:
This version should be incremental, i.e. starting from 1.

Reply:
hmmm then I misunderstood the usage of the version - I thought I can't set it to 1 as there was already an old version released and then I just took 4 as this is hopefully now the release version. Fixed all in one commit

Reply:
When not already defined, it's implicitly 0. So this first update will be from 0 to 1. So there is no correlation to the openHAB version. Hopefully this explains. 🙂